docs/api/ref/Errors.md

```diff
@@ -7,8 +7,43 @@ import TabItem from '@theme/TabItem';
 
 FDC3 API operations may sometimes result in an error, which must be returned to the caller. Errors should be returned by rejecting the promise returned by the API with a JavaScript `Error` object (or equivalent for the language of the implementation). The `Error` Object's message should be chosen from the appropriate Error enumeration below.
 
+## `AgentError`
+
+Contains constants representing the errors that can be encountered when calling the [`getAgent`](getAgent) function to establish connectivity to a Desktop Agent. Primarily used with web applications, but may also be used in other language
+implementations.
+
+<Tabs groupId="lang">
+<TabItem value="ts" label="TypeScript/JavaScript">
+
+```ts
+enum AgentError { 
+    /** Returned if no Desktop Agent was found by any means available or 
+     * if the Agent previously connected to is not contactable on a  
+     * subsequent connection attempt.*/
+    AgentNotFound = "AgentNotFound",
+
+    /** Returned if validation of the app identity by the Desktop Agent 
+     * Failed or the app is not being allowed to connect to the Desktop Agent 
+     * for another reason. */ 
+    AccessDenied = "AccessDenied",
+
+    /** Returned if an error or exception occurs while trying to set  
+     * up communication with a Desktop Agent. */ 
+    ErrorOnConnect = "ErrorOnConnect",
+
+    /** Returned if either the failover function itself, or what it returned,  
+     * was not the right type. */ 
+    InvalidFailover = "InvalidFailover" 
+} 
```

Comment:
AgentError and BridgingError are inconsistent with all the other enums - all the others feature trailing commas, but these ones don't.

Reply:
> ... all the others feature trailing commas, but these ones don't.
Gah, I tend to forget thats the typescript convention (which differs from JS where its an error )

Switched all to double quotes - also in sources.

Fixed in dda1f278fae1ca37f6ba4c41cbbba644ff6ae3ea